code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -404,7 +404,10 @@
 
 				//Actually disarm them
 				for(var/obj/item/I in holding)
-					if(I)
+					if(istype(I,/obj/item/weapon/melee/arm_blade)) // check if its a armblade or not

```

Comment:
This is a really bad way to handle this, by adding a particular exception here, instead of creating a proper framework for items that can't be disarmed. Also, it removes the check to see if I exist, which might cause runtimes in the future.

Reply:
It shouldn't cause runtime unless I is deleted right same moment as you aquired it in for each loop. The loop itself wouldn't return null, iirc.

Reply:
But it's true, better to implement framework.